raiden/network/proxies/token.py

```diff
@@ -19,6 +19,16 @@
 GAS_REQUIRED_FOR_APPROVE = 58792
 
 
+def _insufficient_balance_message(prefix, balance, required):
```

Comment:
This was not my intent when in my previous review.

My intent was that we don't need to generalize this code into a function. Just take the body of the function and add it above where i added "MARKER" in my review.

Reply:
What we're trying to do in the proxies is to stop trying to "generalize" code because what data is checked against and what exception is thrown is different in each of the 3 failure possibilities we have (precondition, gas estimation or transaction failure)